R/calculate_within_batch_ari.R

```diff
@@ -0,0 +1,100 @@
+#' Calculate within-batch ARI scores from a merged SCE object.
+#'
+#'
+#' @param individual_sce_list A list of individual SCE objects
```

Comment:
I might add a bit more here (but you will probably need to fix spacing locally!!)
```suggestion
#' @param individual_sce_list A named list of individual SCE objects. It is assumed these have a reduced dimension slot with principal components named "PCA".
```